### PR TITLE
52 fetch recipes from dashboard

### LINKED ISSUE #52: FETCH recipes from Dashboard

### PR DESCRIPTION
**Pull request by Bokaiii**

### CODE REVIEW

File: src/scripts/start-batch-main.js [hunk 1]
```diff
@@ -3,9 +3,15 @@ import * as recipe from "./ui-elements/recipe.js"
...
-new recipe.recipe('ipa', 150, 94);
-new recipe.recipe('stout', 200, 200);
-new recipe.recipe('ale', 100, 92);
-new recipe.recipe('alcoholFree', 125, 95, 'alcohol free');
+fetch('https://api.bierproductie.nymann.dev/recipes/')
+.then(response => response.json())
+.then(data => {
+    console.log(data);
+    data.forEach(r => {
+        if(r.name === "alcoholFree"){
```
Comment (Kechr) [inline src/scripts/start-batch-main.js:11]: Why check if it is the alcohol free recipe?

Comment (kkmp-dk) [reply src/scripts/start-batch-main.js:11]: When we get the data from API  it is called "alcoholFree", 
so we want to make it say "alcohold Free" in the html instead right?

Comment (kkmp-dk) [reply src/scripts/start-batch-main.js:11]: We can remove it, when the html will just show alcoholFree

Comment (Kechr) [reply src/scripts/start-batch-main.js:11]: Aight, makes sense.

**Pull request merged**